### DISCUSSION TITLE
Support for Deno

### DISCUSSION THREAD

**Discussion by Martinsos** [wasp-lang/wasp#932] [Archived]
Discord user ant2024 asked about this. https://discord.com/channels/686873244791210014/867713251479519232/1059317389512163358

**Comment by baptistefkt** [comment]
+1 with release of deno 2.0, it seems to me that it will become the standard of server-side js. It embraces web standards, is more performant and more secure. It also reduces the complexity in term of DX (no ts-config, no additional linter, formatter, test runner needed).
